app/src/main/java/pub/devrel/easypermissions/sample/MainFragment.java

```diff
@@ -40,6 +42,18 @@ public void onClick(View v) {
     }
 
     @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (requestCode == EasyPermissions.SETTINGS_REQ_CODE) {
```

Comment:
No, resultCode always 0 -> RESULT_CANCELED
Since from settings screen we could only go back. All the user needs to do is checking the permission that may be changed if he was from settings screen.